site/content/includes/m3dbnode/annotated_config.yaml

```diff
@@ -359,6 +359,9 @@ db:
         # Whether or not writes to leaving shards count towards consistency
         # Default = false
         shardsLeavingCountTowardsConsistency: <bool>
+        # Whether or not writes to both leaving and initiazing shards as pair counts toward consistency
+        # Default = false
+        shardsLeavingAndInitiazingCountTowardsConsistency: <bool>
```

Comment:
nit: shardPairsLeavingAndInitializingCountTowardsConsistency

even without that, there is a spelling mistake